packages/falcon-shop-data/src/Customer/CustomerWithAddressesQuery.tsx

```diff
@@ -28,7 +28,27 @@ export const GET_CUSTOMER_WITH_ADDRESSES = gql`
   }
 `;
 
-export class CustomerWithAddressesQuery extends Query<Customer> {
+export type CustomerWithAddressesResponse = {
+  customer: Pick<Customer, 'id' | 'firstname' | 'lastname' | 'email'> & {
+    addresses: Pick<
+      Address,
+      | 'company'
+      | 'firstname'
+      | 'lastname'
+      | 'street'
+      | 'postcode'
+      | 'city'
+      | 'countryId'
+      | 'defaultBilling'
+      | 'defaultShipping'
+      | 'region'
+      | 'regionId'
+      | 'telephone'
+    >;
```

Comment:
`customer.addresses` is an array of `Address`
```suggestion
    >[];
```